sdcard/RES/CAPTURES/THIS IS DEMO FOLDER.TXT

```diff
@@ -0,0 +1,3 @@
+your personal recordes were saved into /USR/CAPTURES
```

Comment:
How about:
```
This is a demo folder. The Capture app now
saves capture files in the /USR/CAPTURES folder.
```

Reply:
I prefer @NotherNgineer text too

Reply:
Thank you very much! 
Suggestion accepted.